classquiz/routers/editor.py

```diff
@@ -127,12 +131,6 @@ async def finish_edit(edit_id: str, quiz_input: QuizInput):
         quiz.background_color = quiz_input.background_color
         quiz.background_image = quiz_input.background_image
         quiz.mod_rating = None
-        for image in images_to_delete:
-            if image is not None:
-                try:
-                    await storage.delete([image])
-                except DeletionFailedError:
-                    pass
```

Comment:
Why remove that?

Reply:
I don't see "images_to_delete" being populated. Am I wrong? It seems it's always set to []

Reply:
It *should* find the images that were in the quiz before it was saved and deletes them, but I'll check that